examples/lock-app/esp32/main/Rpc.cpp

```diff
@@ -0,0 +1,152 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "sdkconfig.h"
+#if CONFIG_ENABLE_PW_RPC
+#include "AppTask.h"
+#include "PigweedLoggerMutex.h"
+#include "RpcService.h"
+#include "button_service/button_service.rpc.pb.h"
+#include "device_service/device_service.rpc.pb.h"
+#include "esp_log.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/event_groups.h"
+#include "freertos/semphr.h"
+#include "freertos/task.h"
+#include "locking_service/locking_service.rpc.pb.h"
+#include "pw_log/log.h"
+#include "pw_rpc/server.h"
+#include "pw_sys_io/sys_io.h"
+#include <support/logging/CHIPLogging.h>
+
+const char * TAG = "RPC";
+
+using chip::DeviceLayer::ConfigurationMgr;
+
+static bool uartInitialised;
+
+extern "C" void __wrap_esp_log_write(esp_log_level_t level, const char * tag, const char * format, ...)
+{
```

Comment:
I see that this is duplicated in multiple examples. Is it possible to move it at a common place?

Reply:
I moved it to the PigweedLogger service in this CL: https://github.com/project-chip/connectedhomeip/pull/7289/files#diff-35995e2a54a1c71a709bdafbcd4f214d7f23c6e196cb995ed0baa7db05ac3d4dR89

Reply:
@dhrishi These changes are already the part of above PR(#7289). I will modify it accordingly when it gets merged.